src/components/LocationPage/LocationPageHeader.tsx

```diff
@@ -124,7 +104,7 @@ const LocationPageHeader = (props: {
           <HeaderSection>
             <LocationCopyWrapper>
               <HeaderTitle isEmbed={isEmbed}>
-                <LocationPageHeading projections={projections} />
+                <LocationPageHeading region={region} />
```

Comment:
Hrm. Is it weird that we're passing in the region instead of using `useRegion()` within the component?  I guess either way is fine. 🤷

Reply:
In general, using React context makes components less reusable and binds them to a specific ... eh ... context. In this example, if we don't pass `region` to `LocationPageHeading`, then the component can only be used inside a `LocationPage` (inside `RegionContext.Provider` to be precise).

My preference is to use React context when:
- components at multiple levels need the same information
- but some components in the middle don't really need them

In this case, `LocationPageHeader` already has the `region`, so it makes sense to just pass it down. It doesn't really matter in this case though because the component is really tailored to this page, but that's the reason.